.github/workflows/build.yml

```diff
@@ -183,6 +183,19 @@ jobs:
             lib/kotlin/cross-test-server/build/install/TestServer/
           retention-days: 3
 
+      - name: Run make precross for swift
+        run: make -C test/swift precross
+
+      - name: Upload swift precross artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: swift-precross
+          if-no-files-found: error
+          path: |
+            test/swift/CrossTests/.build/x86_64-unknown-linux-gnu/debug/TestServer
+            test/swift/CrossTests/.build/x86_64-unknown-linux-gnu/debug/TestClient
+
```

Comment:
actually do you mind pull this out to a separate step other than lib-java-and-kotlin?

Reply:
Done